.env.example

```
@@ -26,14 +26,21 @@ JASA_REDIS_URL=
 JASA_SEARCH_CACHE_TTL_SECONDS=129600
 
-# Grounded snippets. Auto enables grounding only when CEREBRAS_API_KEY is set.
+# Grounded snippets. Auto enables grounding when any waterfall tier has its
+# credential set. The LLM call walks an ordered waterfall of OpenAI-compatible
+# chat-completions endpoints so one rate-limited provider does not discard a
+# page fetch that was already paid for. The chain ships in
+# src/jasa/grounding/waterfall.yaml; point JASA_GROUNDING_WATERFALL_PATH at
+# your own copy to swap providers without rebuilding. The LLM_* values below
+# supply the defaults for any tier that omits them.
 JASA_GROUNDING_MODE=auto
 JASA_GROUNDING_CONCURRENCY=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JASA_GROUNDING_CONCURRENCY key should go before the JASA_GROUNDING_MODE key

(UnorderedKey)